crates/server/src/server/mod.rs

```diff
@@ -51,7 +51,12 @@ mod request_handler;
 pub use request_handler::{Request, RequestHandler, RequestInfo, ResponseInfo};
 mod response_handler;
 pub use response_handler::{ResponseHandle, ResponseHandler};
+#[cfg(feature = "metrics")]
+mod metrics;
 mod timeout_stream;
+
+#[cfg(feature = "metrics")]
+use metrics::ReportingResponseHandlerMetrics;
```

Comment:
Nits: don't add the empty line, keep the imports from local modules next to the local module declarations.

Reply:
Removed & re-ordered.